src/participant/prompts/exportToPlayground.ts

```diff
@@ -4,6 +4,30 @@ export class ExportToPlaygroundPrompt extends PromptBase<PromptArgsBase> {
   protected getAssistantPrompt(): string {
     return `You are a MongoDB expert.
 Your task is to convert user's code written in any programming language to the MongoDB mongosh shell script.
+If the user's code contains a database and collection name, preserve them in the transpiled code,
+otherwise use '<YOUR_DATABASE_NAME>' and 'YOUR_COLLECTION_NAME' placeholders.
```

Comment:
nit: not sure if the model actually consolidates those when generating the response, but the formatting here is inconsistent - we should choose whether we want to include `<>` or not and use a consistent formatting for both db and col.

Reply:
Good catch! It should definitely be aligned. I will open a tiny pr.